.github/workflows/github_server.yml

```diff
@@ -7,6 +7,12 @@ on:
       tags:
         required: False
         description: 'Optional. Use a "tag expression" specify which tagged tests to run (https://cucumber.io/docs/cucumber/api/#tag-expressions)'
+      show_docker_output:
+        required: false
+        default: false
+        type: boolean
+        description: 'Show the docker logs while building the GitHub server container. It will also save the docker log artifact. This might show sensitive config information.'
+      #### Developer note: You probably don't need this
```

Comment:
In our documentation we're considering advising our users to look at this file as an example of how to make their own workflow file. They'll need to customize some things. It may also help them to understand some details.

If they don't look at it, these notes may help us advise them on how to create their own file.